barretenberg/cpp/src/barretenberg/vm/avm_trace/avm_execution.cpp

```diff
@@ -752,6 +752,26 @@ std::vector<Row> Execution::gen_trace(std::vector<Instruction> const& instructio
 
     auto trace = trace_builder.finalize();
     vinfo("Final trace size: ", trace.size());
+    vinfo("Number of columns: ", trace.front().SIZE);
```

Comment:
should we only run this if -v is set?

Reply:
I wrote it in a way that if `vinfo` were a macro, we'd only execute any part of it if `-v` is on. However, it's not a macro exactly: https://github.com/AztecProtocol/aztec-packages/blob/ca584a0c2e611ad2b1a1a3295117328bc2ec7ec5/barretenberg/cpp/src/barretenberg/common/log.hpp#L71

I can try to work around it later, I don't want to add a lot of `if (verbose)` and have to depend on an `extern verbose`. However, the time to calculate all this seems to be negligible.